partner_invoice_target/models/account_invoice.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models,fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    # override vat to add translation
+    vat = fields.Char()
+    is_invoice_target = fields.Boolean(related="partner_id.is_invoice_target", store=True, readonly=True)
```

Comment:
```suggestion
    is_invoice = fields.Boolean("Is Invoice")
```

We can't make it a related field. @rinaldifirdaus Didn't Kanda-san explain this to you? We should extend the method of purchase.order which prepares values for the invoice.